### PR TITLE
update features for v1.2 of Contao Consent

### PR DESCRIPTION
**Pull request by frontendschlampe**

### CODE REVIEW

File: linter/allowlists/default.txt [hunk 1]
```diff
@@ -195,6 +206,8 @@ Stylesheet
 stylesheet
 svg
 Symfony
+tawk
+tawk to
```
Comment (fritzmg) [inline linter/allowlists/default.txt:210]:
```suggestion
```

File: linter/allowlists/default.txt [hunk 2]
```diff
@@ -111,12 +116,15 @@ LASR
 LDAP
 LastLogin
 Leaflet
+leaflet
+leaflet maps
```
Comment (fritzmg) [inline linter/allowlists/default.txt:120]:
```suggestion
```

File: linter/allowlists/default.txt [hunk 3]
```diff
@@ -59,11 +59,14 @@ Eventlist
 Excel
 excel
 Facebook
+facebook
+facebook pixel
```
Comment (fritzmg) [inline linter/allowlists/default.txt:63]:
```suggestion
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — CHANGES_REQUESTED** [review]
`maps`, `pixel` and `to` are regular English words, so imho they should not be on the default allowlist.

Where was "to" flagged?

**Comment by frontendschlampe** [comment]
> maps, pixel and to are regular English words, so imho they should not be on the default allowlist.

you're right, but all words were complained by the linter

> Where was "to" flagged?

It comes from hofff/contao-tawk-to and was complained by the linter, too.

**Comment by fritzmg** [comment]
Oh I see. Oh well 🤷 

// edit: I'll resolve the conflicts for you ;)

**Review by fritzmg — DISMISSED** [review]

**Comment by fritzmg** [comment]
> // edit: I'll resolve the conflicts for you ;)

@frontendschlampe I can't because you did not allow edits for maintainers in your PR (I think). You need to resolve it manually.

**Comment by frontendschlampe** [comment]
> I can't because you did not allow edits for maintainers in your PR (I think).

I don't know ... ;-)

**Review by fritzmg — APPROVED** [review]

**Pull request merged**